policies/optimized-images.md

```diff
@@ -140,71 +173,99 @@ For an `<img>` element, if neither the width or the height of the source image e
   </tr>
 </table>
 
-For an `<img>` element, if neither the width or the height of the source image exceeds the number of pixels allowed by the policy in the container (by default, 2 times of its container's width or height), the image will be rendered correctly; if the height the source image exceeds the limit, the image will be rendered as a placeholder image.
+For an `<img>` element, if neither the intrinsic width or the intrinsic height of the source image exceeds the number of pixels allowed by the policy in the container (2 times larger than the container's width or height), the image will be rendered correctly; if the intrinsic height the source image exceeds the limit, the image will be rendered as a placeholder image
 </br></br>
 
+
 <a name="unoptimized-images">
 
-### "unoptimized-images" policy
+### "unoptimized-*-images" policy
 
 </a>
 
-When optimizing images, the file size should be kept as small as possible. The larger the download size is, the longer it takes a page to load. Stripping metadata, picking a good image format, and using image compression, are all common ways to optimize an image's file size. `unoptimized-images` is a policy controlled feature that restricts images to have a file size (in terms of number of bytes) no more than X times bigger than the image size (width * height) on the web page.
+When optimizing images, the file size should be kept as small as possible. The larger the download size is, the longer it takes a page to load. Stripping metadata, picking a good image format, and using image compression, are all common ways to optimize an image's file size. `unoptimized-images` is a policy controlled feature that restricts images to have a file size (bytes) no more than X times larger than the image resolution (width x height, pixels) on the web page.
 
-When a document is disallowed to use `unoptimized-images` policy, its `<img>` elements whose file sizes are too big will be rendered as placeholder images.
+When a document is disallowed to use `unoptimized-images` policy, its `<img>` elements whose file sizes are too large will be rendered as placeholder images.
+
+We are proposing 3 policies for you to experiment:
+
+*   **"unoptimized-lossy-images"**: any `<img>` element of JPEG format is restricted to not exceed a byte-per-pixel ratio of _***X***_, with a fixed _**1KB**_ overhead allowrance.
```

Comment:
'to not exceed' -> 'so as not to exceed'

Copy this edit to the next two items.